bin/cli-flags.js

```diff
@@ -405,7 +405,7 @@ module.exports = {
       description: 'The public hostname/ip address of the server.',
     },
     {
-      name: 'firewall',
+      name: 'allowed-hosts',
```

Comment:
Can we add tests for `--allowed-hosts` in `cli.test.js`?

Reply:
yep lets add it

Reply:
fixed